server/src/main/java/com/objectcomputing/checkins/services/questions/QuestionResponseDTO.java

```diff
@@ -0,0 +1,31 @@
+package com.objectcomputing.checkins.services.questions;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+
+import javax.validation.constraints.NotBlank;
+import java.util.UUID;
+
+public class QuestionResponseDTO {
```

Comment:
Is this the same as Question, is there a schema difference or something I'm not seeing. Not saying this is a bad idea, just want to know what it is I'm missing.